templates/rhdm70-kieserver-basic-s2i.yaml

```diff
@@ -146,13 +146,13 @@ parameters:
 - displayName: Maven repository username
   description: Username to access the Maven repository.
   name: MAVEN_REPO_USERNAME
-  value: ''
-  required: false
+  required: true
```

Comment:
according with the PR that David sent, it have default values if empty, I think it could be a not required field.

Reply:
IMO it is better when using a template to make sure of what will be used in the container so that by inspecting the ENV variables you can see/controll all the credentials that have been generated, the same way as the kie admin and controllerUser.

Reply:
@errantepiphany the default value is below